api/client/client.go

```diff
@@ -294,18 +319,23 @@ func (srv *server) getResultMap(jsonData []byte, target string) (result map[stri
 }
 
 // request performs the common logic for Sign and Info, performing the actual
-// request and returning the resultant certificate.
-func (srv *server) request(jsonData []byte, target string) ([]byte, error) {
+// request and returning the resultant certificate or bundle.
+func (srv *server) request(jsonData []byte, target string, returnBundle bool) ([]byte, error) {
 	result, err := srv.getResultMap(jsonData, target)
 	if err != nil {
 		return nil, err
 	}
-	cert := result["certificate"].(string)
+
+	key := "certificate"
+	if returnBundle {
+		key = "bundle"
+	}
+	cert := result[key].(string)
 	if cert != "" {
 		return []byte(cert), nil
 	}
 
-	return nil, errors.Wrap(errors.APIClientError, errors.ClientHTTPError, stderr.New("response doesn't contain certificate."))
+	return nil, errors.Wrap(errors.APIClientError, errors.ClientHTTPError, stderr.New("response doesn't contain certificate/bundle."))
```

Comment:
It would be cleaner to use the content of the  "key" variable in the message.

Reply:
Thanks for pointing this out. I did actually miss a bigger change here.